rector.php

```diff
@@ -14,7 +15,7 @@
 
     // Define what rule sets will be applied
     $parameters->set(Option::SETS, [
-        SetList::EARLY_RETURN,
+        // SetList::EARLY_RETURN,
```

Comment:
Die Rule findet noch viel. Und an vielen Stellen fände ich die Anpassung keine Verbesserung.
Deswegen erstmal deaktiviert, muss man sich dann in Ruhe mal anschauen, wie man damit weiter verfährt.